AnswerContainer.h

```diff
@@ -1,18 +1,19 @@
 #pragma once
 #include <cpprest/http_listener.h>
 #include <cpprest/json.h>
-#include "irequests.h"
+#include "IRequests/IRequests.h"
 #include "MSSQLDatabase.h"
+#include "AnswerContainerInterface.h"
 using namespace web;
 using namespace web::http;
 using namespace web::http::experimental::listener;
 
-class answercontainer {
+class answercontainer: public answercontainerinterface{

```

Comment:
It seems to me that it is better to call it IAnswerContainer instead of answercontainerinterface.